clients/include/testing_scal.hpp

```diff
@@ -28,17 +28,12 @@ void testing_scal_bad_arg()
 
     rocblas_status status;
 
-    std::unique_ptr<rocblas_test::handle_struct> unique_ptr_handle(new rocblas_test::handle_struct);
-    rocblas_handle handle = unique_ptr_handle->handle;
+    rocblas_local_handle handle;
 
     rocblas_int size_x = N * incx;
 
     // allocate memory on device
-    auto dx_managed = rocblas_unique_ptr{rocblas_test::device_malloc(sizeof(T) * size_x),
-                                         rocblas_test::device_free};
-    return;
-
```

Comment:
I'm not sure why this return is here. I'm removing it assuming it was accidentally checked in after someone was hand-debugging code and put it in there as an early return to test what was causing a failure.